vehicle_controller:
  ros__parameters:
    v_max: 10.0
    a_lat_max: 3.0
    speed_rate_limit: 2.0

    lookahead_min: 2.0
    lookahead_max: 12.0
    lookahead_speed_gain: 0.8
    lookahead_curv_gain: 8.0

    pid_kp: 1.2
    pid_ki: 0.1
    pid_kd: 0.05
    pid_i_min: -3.0
    pid_i_max: 3.0
